docs/compass/04-01-installation.md

```diff
@@ -1,78 +1,68 @@
+          # The Namespace of the currently used gateway
+          global.istio.gateway.namespace: "kyma-system"
+          global.connector.secrets.ca.name: "connector-service-app-ca"
+          # The Namespace with a Secret that contains a certificate for the Connector Service
+          global.connector.secrets.ca.namespace: "kyma-integration"
+          # The parameter that enables the Compass gateway, as the default Kyma gateway is disabled in this installation mode
+          gateway.gateway.enabled: "false"
+          global.disableLegacyConnectivity: "true"
+    ```
 
+3. Install Compass. ​There are three possible installation options:
 
-### Kyma Compass
+    | Installation option     	| Value to use in the installation command   	| Example value          	|
+    |-------------------------	|-------------------	|-------------------------	|
+    | From the `master` branch 	| `master`          	| `master`                	|
+    | From the specific commit on the `master` branch 	| `master-{COMMIT_HASH}` 	| `master-34edf09a` 	|
+    | From the specific PR       	| `PR-{PR_NUMBER}`         	| `PR-1420`     	|
 
-This is a multi-tenant and multi-Runtime mode that provides a cluster with Compass and only the selected Kyma components that Compass uses. To enable this mode, create this ConfigMap and then perform the cluster Kyma installation using the
- [`installer-cr-cluster-compass.yaml.tpl`](https://github.com/kyma-project/kyma/blob/master/installation/resources/installer-cr-cluster-compass.yaml.tpl) configuration file:
-
-```yaml
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: compass-overrides
-  namespace: kyma-installer
-  labels:
-    installer: overrides
-    component: compass
-    kyma-project.io/installation: ""
-data:
-  # The parameter that enables the Compass gateway, as the default Kyma gateway is disabled in this installation mode
-  gateway.gateway.enabled: "true"
-  # The name of the currently used gateway
-  global.istio.gateway.name: compass-istio-gateway
-  # The Namespace of the currently used gateway
-  global.istio.gateway.namespace: compass-system
-  # The parameter that disables preconfiguration for the Compass Runtime Agent
-  global.agentPreconfiguration: "false"
-  # The Namespace with a Secret that contains a certificate for the Connector Service
-  global.connector.secrets.ca.namespace: compass-system
+    Once you decide on the installation option, use this command:
+    ```bash
+    kubectl apply -f https://storage.cloud.google.com/kyma-development-artifacts/compass/{INSTALLATION_OPTION}/compass-installer.yaml
+    ```
+​
+4. Check the installation progress. To do so, download the script that checks the progress of the installation:
+```bash
+wget https://storage.cloud.google.com/kyma-development-artifacts/compass/{INSTALLATION_OPTION}/is-installed.sh && chmod +x ./is-installed.sh
+```
+​
+Then, use the script to check the progress of the Compass installation:
+```bash
+./is-installed.sh
```

Comment:
./is-installed.sh - please point where that file exists, like: ./installation/cmd/run.sh

Reply:
The script is downloaded from the bucket using the wget command and then, by means of the second command, we use it. So in other words, it can be placed wherever.